packages/web/src/js/__tests__/FileUploader.test.ts

```diff
@@ -0,0 +1,65 @@
+import { clearFixture, getFixture } from '../../../tests/helpers/fixture';
+import FileUploader from '../FileUploader';
+
+describe('FileUploader', () => {
```

Comment:
I would love to see some tests of configurable data attributes (eg max file size)

Reply:
And could you please give me these AI generated tests?